### PR TITLE
Document additions in the 4.2.0 RC

### PR DESCRIPTION
**Pull request by linuswillner**
Prepare the documentation for the release of version 4.2.0. Quality control requested from the PR developers (SteamingMutt, zaza7, thenightex, Nenkai).

### CODE REVIEW

**Review by Nenkai — COMMENTED** [review]
Seems to cover everything that was pushed into experimental. +1

**Review by Nenkai — APPROVED** [review]

**Review by zaza7 — CHANGES_REQUESTED** [review]
It appears we're still missing an updated piece of playlist which is "delete/remove all" to well clear the playlist.

I take that back. I don't like the command. I'd rather it be `playlist clear`

**Review by zaza7 — APPROVED** [review]

File: docs/install_windows.md [hunk 1]
```diff
@@ -139,6 +139,24 @@ deleteTimeout | The amount of time after which to delete the messages. | Insert
...
+**IMPORTANT NOTE:** Unless you use Bezerk, the WildBeast WSM, **do not modify anything here as it's for advanced users only!**
+  
+Property | Explanation | Notes |
+-------- | ----------- | ----- |
+use | Whether Bezerk shall be used or not. | Do not modify unless you use Bezerk. |
+uri | Specifies the websocket address for the Bezerk server. | As above. Default is `ws://localhost:1337` |
+  
+###Elasticsearch section
+  
+**IMPORTANT NOTE:** Unless you use Elastic and Kibana like we do, **don't modify this unless your instance is and want easier log stashing!**
```
Comment (Dougley) [inline docs/install_windows.md]: Akward wording

**Review by Dougley — CHANGES_REQUESTED** [review]

File: docs/install_linux.md [hunk 2]
```diff
@@ -179,13 +179,31 @@ user | RethinkDB user account to use for accessing the database. | Admin by defa
...
+-------- | ----------- | ----- |
+prefix | The command prefix the bot will use. | Can be a special character like `/` etc. |
+autodeletemsg | Automatically delete music-related messages after a while. | The time for this is possible to configure. Boolean. |
+deleteTimeout | The amount of time after which to delete the messages. | Insert time in milliseconds, default 3000. |
+deleteTimeoutLong | Same as above, but for messages that have a longer timeout.  | For now only [this message](https://github.com/TheSharks/WildBeast/blob/master/runtime/internal/voice.js#L204) uses the long timeout. In milliseconds, default 6000. |
+maxvcslots | How many concurrent voice connections the bot can have until it won't join more. | Default limit is 10, depends on the beefyness of your system. |
+  
+###Bezerk section
+  
+**IMPORTANT NOTE:** Unless you use Bezerk, the WildBeast WSM, **do not modify anything here as it's for advanced users only!**
```
Comment (SteamingMutt) [inline docs/install_linux.md]: Akward wording

File: docs/install_windows.md (hunk 1 above)
Comment (linuswillner) [reply docs/install_windows.md]: That's a typo...

**Comment by linuswillner** [comment]
@SteamingMutt The above has been resolved in https://github.com/TheSharks/WildBeastDocs/pull/17/commits/636ee981030a1ef9a45b3c1bf87af296e3b543e5

**Review by Dougley — APPROVED** [review]

**Review by thenightex — APPROVED** [review]
gw

**Comment by linuswillner** [comment]
Seems good. Just need to fix blocking issue https://github.com/TheSharks/WildBeastDocs/issues/18.

**Comment by linuswillner** [comment]
Docs ready for launch. Going to standby mode for the update push.

**Pull request merged**